spot-ml/src/test/scala/org/apache/spot/lda/SpotLDAWrapperTest.scala

```diff
@@ -65,30 +62,34 @@ class SpotLDAWrapperTest extends TestingSparkContextFlatSpec with Matchers {
     val logger = LogManager.getLogger("SuspiciousConnectsAnalysis")
     logger.setLevel(Level.WARN)
 
-    val ldaAlpha =  1.2
-    val ldaBeta = 1.001
-    val ldaMaxIterations = 20
+    val ldaAlpha = 1.002
```

Comment:
It's leftover of my testing of training with EM and scoring with Online, will revert to the original value, thanks!